src/quo/components/community/community_detail_token_gating/view.cljs

```diff
@@ -0,0 +1,69 @@
+(ns quo.components.community.community-detail-token-gating.view
+  (:require [clojure.string :as string]
+            [quo.components.community.community-detail-token-gating.style :as style]
+            [quo.components.dividers.divider-label.view :as divider-label]
+            [quo.components.markdown.text :as text]
+            [quo.components.tags.collectible-tag.view :as collectible-tag]
+            [quo.components.tags.token-tag.view :as token-tag]
+            [react-native.core :as rn]
+            [utils.i18n :as i18n]))
+
+(defn- token-view
+  [{:keys [collectible? img-src amount sufficient?] :as token}]
+  (if collectible?
+    [collectible-tag/view
+     {:collectible-name    (:symbol token)
+      :size                :size-24
+      :collectible-img-src img-src
+      :options             (when sufficient?
+                             :hold)}]
+    [token-tag/view
+     {:token-symbol  (:symbol token)
+      :size          :size-24
+      :token-value   amount
+      :token-img-src img-src
+      :options       (when sufficient? :hold)}]))
+
+(defn- tokens-row
+  [{:keys [tokens divider? first?]}]
+  [:<>
+   [rn/view
+    {:style (style/token-row first?)}
+    (map-indexed (fn [token-index token]
+                   ^{:key (str "token-" token-index)}
```

Comment:
Index might not be the best way to key a list. Perhaps a more unique value from the `token` map should be used

Reply:
The db doesn't have any unique value for tokens here. Also, the chances of the items changing after rendering are very rare, so I thought it is okay to use the index here.